addons/l10n_in_hr_payroll/report/payslip_report_view.xml

```diff
@@ -84,7 +84,7 @@
         <field name="groups_id" eval="[(4,ref('hr_payroll.group_hr_payroll_user'))]"/>
         <field name="arch" type="xml">
             <data>
-                <xpath expr="//div[contains(@class, 'manage_reports')]" position="inside">
+                <xpath expr="//div[hasclass('manage_reports')]" position="inside">
```

Comment:
Isn't this a case where `@contains` was the intended behavior? I see the [parent view](https://github.com/odoo/odoo/blob/4c6c16a38246727e3a32ba67b8aa3abd1aec6d5c/addons/hr/views/hr_views.xml#L551) has a "o_kanban_manage_reports" class, but no "manage_reports" one. Turn it into a `hasclass('o_kanban_manage_reports')`?